src/pybind/mgr/dashboard/services/orchestrator.py

```diff
@@ -189,6 +219,7 @@ def __init__(self):
         self.services = ServiceManager(self.api)
         self.osds = OsdManager(self.api)
         self.daemons = DaemonManager(self.api)
+        self.upgrades = UpgradeManger(self.api)
```

Comment:
```suggestion
        self.upgrades = UpgradeManager(self.api)
```

Reply:
?

Reply:
Ok I see spell check. Thanks for noticing